# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1281302490 25200
#      Sun Aug 08 14:21:30 2010 -0700
# Node ID 5502105b4f175efac896e60ef1dd1a042fea6d1a
# Parent  045558999570cb8c4e888c93baf5162e86963c8f
Run do_subst macro quietly to avoid screen clutter during build.

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,13 @@
+2010-08-08  Rik <octave@nomad.inbox5.com>
+
+	* Makefile.am: Run do_subst macro quietly without cluttering screen 
+	during build.
+
 2010-08-08  Ben Abbott <bpabbott@mac.com>
 
 	* plot/print.m: Include 'colormap' when converting RGB to mono.
 
 2010-08-07  Ben Abbott <bpabbott@mac.com>
 
 	* plot/__fltk_print__.m: When running pstoedit, crop eps file after it
 	is created. Correctly check error status.
diff --git a/scripts/Makefile.am b/scripts/Makefile.am
--- a/scripts/Makefile.am
+++ b/scripts/Makefile.am
@@ -323,17 +323,17 @@ gethelp_SOURCES = gethelp.cc
 		cp $(srcdir)/DOCSTRINGS DOCSTRINGS; \
 		touch -r $(srcdir)/DOCSTRINGS DOCSTRINGS; \
 	fi
 	$(srcdir)/mkdoc "$(srcdir)" $(FCN_FILES) -- $(GEN_FCN_FILES) > $@
 	$(top_srcdir)/move-if-change $@ DOCSTRINGS
 	touch $@
 
 $(GEN_FCN_FILES) : %.m : %.in Makefile
-	$(do_subst_config_vals)
+	@$(do_subst_config_vals)
 
 check-m-sources:
 	@echo -n "checking whether files in source tree are listed in module.mk files..."; \
 	for f in $$(find $(srcdir) -name '*.m'); do \
 	  found=false; \
 	  for m in $(FCN_FILES); do \
 	    if [ "$$f" = $(srcdir)/"$$m" ]; then \
 	      found=true; \
diff --git a/src/ChangeLog b/src/ChangeLog
--- a/src/ChangeLog
+++ b/src/ChangeLog
@@ -1,8 +1,13 @@
+2010-08-08  Rik <octave@nomad.inbox5.com>
+
+	* Makefile.am: Run do_subst macro quietly without cluttering screen 
+	during build.
+
 2010-08-08  Shai Ayal  <shaiay@users.sourceforge.net>
 
 	* DLD-FUNCTIONS/fltk_backend.cc
 	(base_graphics_backend::print_figure): Add a redraw to get the
 	thing going.
 
 2010-08-05  Shai Ayal  <shaiay@users.sourceforge.net>
 
diff --git a/src/Makefile.am b/src/Makefile.am
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -738,20 +738,20 @@ endif
 $(OPT_INC) : %.h : %.in
 	$(MAKE) -C $(@D) $@
 
 ## defaults.h and oct-conf.h must depend on Makefile.  Calling configure
 ## may change default/config values.  However, calling configure will also
 ## regenerate the Makefiles from Makefile.am and trigger the rules below.
 
 defaults.h: defaults.h.in Makefile
-	$(do_subst_default_vals)
+	@$(do_subst_default_vals)
 
 oct-conf.h: oct-conf.h.in Makefile
-	$(do_subst_config_vals)
+	@$(do_subst_config_vals)
 
 oct-errno.cc: oct-errno.cc.in Makefile
 	if test -n "$(PERL)"; then \
 	  $(srcdir)/mk-errno-list --perl "$(PERL)" < $< > $@-t; \
 	elif test -n "$(PYTHON)"; then \
 	  $(srcdir)/mk-errno-list --python "$(PYTHON)" < $< > $@-t; \
 	else \
 	  $(SED) '/@SYSDEP_ERRNO_LIST@/D' $< > $@-t; \
